shell/platform/windows/system_utils_winuwp.cc

```diff
@@ -23,12 +26,11 @@ std::vector<LanguageInfo> GetPreferredLanguageInfo() {
 
 std::vector<std::wstring> GetPreferredLanguages() {
   std::vector<std::wstring> languages;
-  // TODO(clarkezone) need to implement a complete version of this function in
-  // order to get full list of platform languages
-  // https://github.com/flutter/flutter/issues/74156
-  languages.push_back(L"en-US");
-  languages.push_back(L"en");
-
+  auto platform_langueages = winrt::Windows::System::UserProfile::
```

Comment:
You misspelled languages here

Reply:
Sorry. I'm little busy now so I'll fix it with new PR by tomorrow.